app/javascript/src/components/Dashboard/Notes/Pane/Form/index.jsx

```diff
@@ -8,51 +8,87 @@ import { TAGS } from "./constants";
 
 import { NOTES_FORM_VALIDATION_SCHEMA } from "../../constants";
 
-const Form = ({ onClose, refetch, note, isEdit }) => {
-  const handleSubmit = values => {
-    // eslint-disable-next-line no-console
-    console.log(values);
-    refetch;
+const Form = ({ onClose, note, isEdit, setShowPane, notes, setNotes }) => {
+  const handleSubmit = ({ title, description, assignedContact, tag }) => {
+    const updatedNotes = [
+      ...notes,
+      {
+        title,
+        description,
+        assignedContactId: assignedContact.value,
+        tag: tag.value,
+      },
+    ];
+    setNotes(updatedNotes);
+    setShowPane(false);
   };
 
+  const contactList = [
+    {
+      id: 1,
+      name: "Elma Herring",
+      email: "elmaherring@unq.com",
+      phone: "+1 (913) 497-2020",
+    },
+    {
+      id: 2,
+      name: "Knapp Berry",
+      email: "knappberry@unq.com",
+      phone: "+1 (951) 472-2967",
+    },
+  ];
+
   return (
     <Formik
       initialValues={note}
       validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
       onSubmit={handleSubmit}
     >
-      {({ isSubmitting }) => (
+      {() => (
         <FormikForm className="w-full">
           <Pane.Body className="space-y-6">
             <Input
               required
               className="w-full flex-grow-0"
               label="Title"
               name="title"
+              placeholder="Enter Title"
             />
             <Textarea
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
+              placeholder="Enter Note Description"
             />
-            <Select
-              isMulti
-              className="w-full"
-              label="Tags"
-              name="ValueList"
-              placeholder="Select tags - You can choose multiple"
-              options={TAGS.map(tag => ({
-                label: tag,
-                value: tag,
-              }))}
-            />
+            <div className="w-full space-y-0">
+              <Select
+                className="w-full"
+                label="Assign To"
+                name="assignedContact"
+                placeholder="Select Role"
+                options={contactList.map(contact => ({
+                  label: contact.name,
+                  value: contact.id,
+                }))}
```

Comment:
Wouldn't it be better to hardcode the labels and values in the constants file? This would improve readability and make the component free of non-visual logic.